.kube-linter.yaml

```diff
@@ -1,3 +1,2 @@
 checks:
   exclude:
-  - "no-extensions-v1beta"
```

Comment:
do we still need `.kube-linter.yaml` after that?

Reply:
I'd like to keep it for later additions

Reply:
It goes slightly against YAGNI principle; when you don't need functionality anymore - you delete it, makes things simple and uncluttered. We can add `.kube-linter.yaml` later, once we really need it.